src/lxc/tools/lxc_top.c

```diff
@@ -273,6 +279,60 @@ static uint64_t stat_match_get_int(struct lxc_container *c, const char *item,
 	return val;
 }
 
+/*
+examples:
+	blkio.throttle.io_serviced
+	8:0 Read 4259
+	8:0 Write 835
+	8:0 Sync 292
+	8:0 Async 4802
+	8:0 Total 5094
+	Total 5094
+
+	blkio.throttle.io_service_bytes
+	8:0 Read 110309376
+	8:0 Write 39018496
+	8:0 Sync 2818048
+	8:0 Async 146509824
+	8:0 Total 149327872
+	Total 149327872
+*/
+static void stat_get_blk_stats(struct lxc_container *c, const char *item,
+			      struct blkio_stats *stats) {
+	char buf[4096];
+	int i,len;
+	char **lines, **cols;
+
+	len = c->get_cgroup_item(c, item, buf, sizeof(buf));
+	if (len <= 0) {
+		ERROR("unable to read cgroup item %s", item);
+		return;
+	}
+
+	lines = lxc_string_split_and_trim(buf, '\n');
+	if (!lines)
+		return;
+
+	memset(stats, 0, sizeof(struct blkio_stats));
+	for (i = 0; lines[i]; i++) {
+		cols = lxc_string_split_and_trim(lines[i], ' ');
+		if (!cols)
+			goto out;
+		if (strcmp(cols[1], "Read") == 0) {
+			stats->read += strtoull(cols[2], NULL, 0);
+		} else if (strcmp(cols[1], "Write") == 0) {
+			stats->write += strtoull(cols[2], NULL, 0);
+		}
+		if (strcmp(cols[0], "Total") == 0) {
+			stats->total = strtoull(cols[1], NULL, 0);
+		}
```

Comment:
Please remove all the brackets around single-lin `if` statements and `if-else-if` ladders. So:

```
if (bla)
        something
else if
        something else
```